lib/index.js

```diff
@@ -30,11 +30,11 @@ var el = new ExecLimiter(Deffy(parseInt(process.env.GPM_PROCESS_COUT_LIMIT) || 1
  *
  * @name Gpm
  * @function
- * @param {String} name The git url or the npm package name.
+ * @param {(String|String[])} packages The git url, the npm package name or a list of them.
```

Comment:
I was using `{String|Array}` -- it's obvious that it's an array of strings. :smile: